packages/components/psammead-image/src/index.jsx

```diff
@@ -1,12 +1,25 @@
 import React from 'react';
 import { number, oneOfType, string } from 'prop-types';
-import styled from 'styled-components';
+import styled, { keyframes } from 'styled-components';
 
 export { default as AmpImg } from './index.amp';
 
+const fadeIn = keyframes`
+  from {
+    opacity: 0;
+  }
+
+  to {
+    opacity: 1;
+  }
+`;
+
 const StyledImg = styled.img`
   display: block;
   width: 100%;
+  opacity: 1;
+  animation-duration: 2s;
```

Comment:
2s seems too long, but love the implementation 😍

Reply:
Yep, should be about 200ms, but this was just to demonstrate it working 👍